README.md

```diff
@@ -1,4 +1,10 @@
 Secure Frame (SFrame)
+[![build](https://img.shields.io/github/actions/workflow/status/goto-opensource/secure-frame-rs/ci.yml?branch=main)](https://github.com/goto-opensource/secure-frame-rs/actions?query=workflow%3A"Continuous+Integration")
+[![version](https://img.shields.io/crates/v/sframe)](https://crates.io/crates/sframe/)
+[![Crates.io](https://img.shields.io/crates/d/sframe)](https://crates.io/crates/sframe)
+[![license](https://img.shields.io/crates/l/sframe.svg?style=flat)](https://crates.io/crates/sframe/)
+[![documentation](https://img.shields.io/badge/docs-latest-blue.svg)](https://docs.rs/sframe/)
+![maintenance](https://img.shields.io/maintenance/yes/2023)
 =======================
```

Comment:
did you intend to put this into the `<h1>` header?

you should probably put this under the `=======`

Reply:
Good catch, was not on purpose^^